From 1335b1b1aff61167f8648f7cb3c569764031960d Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 24 Jan 2022 11:37:06 +0000
Subject: [PATCH] meson: drop unused argument for i18n.merge_file()

Ignored in Meson < 0.60.0, deprecated since 0.60.1 and fatal since 0.61.0.

data/appdata/meson.build:5:0: ERROR: Function does not take positional arguments.
data/meson.build:16:0: ERROR: Function does not take positional arguments.
---
 data/appdata/meson.build | 1 -
 data/meson.build         | 1 -
 2 files changed, 2 deletions(-)

diff --git a/data/appdata/meson.build b/data/appdata/meson.build
index 1f5744e..d2c2c3a 100644
--- a/data/appdata/meson.build
+++ b/data/appdata/meson.build
@@ -3,7 +3,6 @@ metainfo_conf.set('app-id', application_id)
 metainfo_conf.set('gettext-package', gettext_package)
 
 metainfo_file = i18n.merge_file(
-  'metainfo-file',
   input: configure_file(
   	input: 'org.gnome.SoundRecorder.metainfo.xml.in.in',
   	output: '@BASENAME@',
diff --git a/data/meson.build b/data/meson.build
index 44117ad..0ff8fa5 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -14,7 +14,6 @@ install_data(
 desktop_conf = configuration_data()
 desktop_conf.set('app-id', application_id)
 desktop_file = i18n.merge_file(
-  'desktop',
   input: configure_file(
   	input: 'org.gnome.SoundRecorder.desktop.in.in',
   	output: '@BASENAME@',
-- 
GitLab

